services/minespace-web/src/components/common/DocumentTable.tsx

```diff
@@ -147,7 +147,7 @@ export const DocumentTable: FC<DocumentTableProps> = ({
       label: FileOperations.View,
       icon: <FileOutlined />,
       clickFunction: (_event, record: MineDocument) =>
-        openDocument(record.document_manager_guid, record.mine_document_guid),
+        openDocument(record.document_manager_guid, record.document_name),
```

Comment:
Ahhhh looks like our other two instances of DocumentTable also pass in the guid as second arg. Probably want to get those too.

Reply:
Yup Fixed the other instances in this PR yesterday: https://github.com/bcgov/mds/pull/2895/files - just failed to include this one 😓